components/shapediver/viewport/buttons/HistoryMenuButton.tsx

```diff
@@ -34,7 +34,8 @@ export default function HistoryMenuButton({
 		const {modelStateId} = await createModelState(
 			undefined, // <-- parameterNamesToInclude: use default according to the theme
 			undefined, // <-- parameterNamesToExclude: use default according to the theme
-			true, // <-- includeImage,
+			false, // <-- includeImage,
```

Comment:
@MajorMeerkatThe3rd I also suggest to keep `includeImage` set to `true` in this case